administrator-manual/en/backup.rst

```diff
@@ -458,6 +458,10 @@ For example, to backup a software installed inside :file:`/opt` directory, add t
 
   /opt/mysoftware
 
+
+The same syntax applies to configuration backup. Modifications should be done inside the file :file:`/etc/backup-config.d/custom.include`.
```

Comment:
We already have a "Configuration backup customization" in this page. Are you sure this is necessary?

Reply:
I think it could be useful to explicitly clarify that there are two different fragments for the _inclusions_ and the _exclusions_